lib/omniauth/strategies/defence_request.rb

```diff
@@ -44,9 +45,12 @@ def full_raw_info_url
       end
 
       def log_request
-        return unless ENV["OAUTH_DEBUG"]
+        return unless ENV["AUTH_DEBUG"]
 
-        log :debug, "curl -H \"Accept: application/json\" -H \"Authorization: Bearer #{access_token.token}\" #{full_raw_info_url}"
+        require 'pp'
```

Comment:
It's required there so its on a need basis - only require pp if we need it as it has a large overhead